frame/system/src/lib.rs

```diff
@@ -573,6 +542,40 @@ decl_module! {
 	}
 }
 
+impl<T: Trait> Module<T> {
+	/// Determine whether or not it is possible to update the code.
```

Comment:
Please move to the other impl block: https://github.com/paritytech/substrate/blob/b3136234c9b41d630670ba8bb8f647ced214d7f2/frame/system/src/lib.rs#L702